lib/util.js

```diff
@@ -89,6 +89,25 @@ function normalize(aPath) {
   var isAbsolute = exports.isAbsolute(path);
 
   var parts = path.split(/\/+/);
```

Comment:
nit: the call to `path.split()` hasn't been removed yet ;)

Reply:
D'oh